design-patterns/gc-intro/_base.scss

```
@@ -0,0 +1,5 @@
+/* REMOVE ONCE IMPLEMENTED IN WET-BOEW */
+.bg-half-width-right {
```

As I proposed earlier

```suggestion
.bg-half-right {
```

And let also add the `bg-half-left` to have a complete utilities suite.

---

Not needed anymore.